test/test_images/probe_helper/probe/handlers/ping.go

```diff
@@ -70,6 +70,7 @@ func (p *PingSourceProbe) Forward(ctx context.Context, event cloudevents.Event)
 	p.EventTimes.RLock()
 	defer p.EventTimes.RUnlock()
 
+	logging.FromContext(ctx).Infow("Checking last observed PingSource tick")
```

Comment:
Nit: seems like this could also just be `logging.FromContext(ctx).Info(...)`?

Reply:
Done.